pages/linux/nmon.md

```diff
@@ -0,0 +1,15 @@
+# nmon
+
+> A system administrator, tuner, benchmark tool.
+
+- Start nmon:
+
+`nmon`
+
+- Save records to file (default args "-s300 -c288" for 300 seconds between refreshing the screen and 288 refreshes):
+
+`nmon -f`
+
+- Save records to file with 30 seconds refreshing gap and 240 refreshes:
```

Comment:
What does "240 refreshes" mean ? Does it mean that within a single interval, it will take 240 measurements ?

Reply:
No. It means the system will take 240 measurements and the gap between each measurement is 30 seconds. So the whole time monitored is 2 hours.
I've drew a illustration to explain this command:

![selection_200](https://user-images.githubusercontent.com/15963141/34199009-37aef102-e5a7-11e7-8b55-a247a8da205e.png)

Reply:
I see. In that case, I would slightly change the description to something like - 
`Save records to file with a total of 240 measurements by taking 30 seconds between each measurement.`

Reply:
Great, I love this sentence!